client/src/components/utils/Login/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import logInSchema from '../../../validation/login';
+import Button from '../Button/index';
+import background from '../../../assets/images/login.background.png';
+import waiter from '../../../assets/images/waiter.png';
+import './style.css';
+
+export default class Login extends React.Component {
+  state = {
+    tableNumber: '',
+    secret: '',
+    err: null,
+  };
+
+  setTableNumber = e => {
+    this.setState({ tableNumber: e.target.value });
+  };
+
+  setSecret = e => {
+    this.setState({ secret: e.target.value });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { tableNumber, secret } = this.state;
+    const { history } = this.props;
+    logInSchema
+      .validateAsync({ tableNumber, secret })
+      .then(() => {
+        return fetch('/api/v1/login', {
+          method: 'POST',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify({
+            tableNumber,
+            secret,
+          }),
+        });
+      })
+      .then(res => {
+        if (res.status === 200) {
+          res.json();
+          history.push('/');
```

Comment:
Before redirecting to the home page update the state for the app component adding the table number to it. To be accessible for all components